src/Routes.js

```diff
@@ -13,28 +14,58 @@ const Dashboard = lazy(() =>
 
 const ActivationKeys = lazy(() => import('./Components/ActivationKeys'));
 
-export const Routes = () => {
+const SuspenseWrapped = ({ children }) => (
+  <Suspense
+    fallback={
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    }
+  >
+    {children}
+  </Suspense>
+);
+
+const AppRoutes = () => {
   const keyDetailsIsEnabled = useFeatureFlag(
     'sed-frontend.activationKeysDetailsPage'
   );
   return (
-    <Suspense
-      fallback={
-        <Bullseye>
-          <Spinner />
-        </Bullseye>
-      }
-    >
-      <Authentication>
-        <Switch>
-          {keyDetailsIsEnabled && (
-            <Route path={paths.activationKey} component={ActivationKey} />
-          )}
-          <Route path={paths.activationKeys} component={ActivationKeys} />
-
-          <Route path={paths.connector} component={Dashboard} />
-        </Switch>
-      </Authentication>
-    </Suspense>
+    <Authentication>
+      <Routes>
+        {keyDetailsIsEnabled && (
+          <Route
+            path={paths.activationKey}
+            element={
+              <SuspenseWrapped>
```

Comment:
ActivationKey was not wrapped by Suspense before. Do we really need to wrap it now with the router upgrade?

Reply:
I do not see this having issues. Rather all pages behave the same on load with a loader.

Reply:
@mkholjuraev Can the `SuspenseWrapped` be the root of this component, like `Suspense` and not wrap each component in `element` separately?